catalog/dags/data_refresh/distributed_reindex.py

```diff
@@ -274,6 +279,24 @@ def drop_connection(worker_conn: str):
     session.commit()
 
 
+@task(trigger_rule=TriggerRule.NONE_FAILED)
+def report_reindexing_status():
```

Comment:
If we really only need this for grouping, maybe we should use an [`EmptyOperator`](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/empty/index.html) instead? According to the docs, it might be more performant:

> It can be used to group tasks in a DAG. The task is evaluated by the scheduler but never processed by the executor.